File: MODUL 5/C/PRAK502-2310817220011-ReginaSilvaMaharatini.c
#include <stdio.h>
#include <math.h>

int hitung(int nilai1, int nilai2){ //Lengkapi Function ini
int hasil = nilai1 - nilai2;
if (hasil < 0) {
    return hasil *= -1;
} else {
    return hasil;
}
}

int mutlak(int angka){ //Lengkapi Function ini
if (angka < 0) {
    return angka *= -1;
}
}

int main()
{
    int a,b,c,d;
 
 scanf("%d",&a);
 scanf("%d",&c);
 scanf("%d",&b);
 scanf("%d",&d);

 int Hasil = hitung(a,b) + hitung(c,d);
 printf("%d",mutlak(Hasil));

 return 0;
}

File: MODUL 5/Python/PRAK502-2310817220011-ReginaSilvaMaharatini.py
def hitung(nilai1, nilai2):
    hasil = nilai1 - nilai2
    if hasil < 0:
        return -hasil
    else:
        return hasil

def mutlak(angka):
    if angka < 0:
        return -angka
    else:
        return angka
    
for i in range(3):
    a, c, b, d = map(int, input().split())
    hasil = hitung(a, b) + hitung(c, d)
    
    print(mutlak(hasil))
    print ("\n")